src/sagas/helpers.js

```diff
@@ -21,7 +21,7 @@ import {
   setIsShowingPinScreen,
   types,
 } from '../actions';
-import { FEATURE_TOGGLE_DEFAULTS, INITIAL_TOKENS, PUSH_NOTIFICATION_FEATURE_TOGGLE, WALLET_SERVICE_FEATURE_TOGGLE, WALLET_SERVICE_REQUEST_TIMEOUT } from '../constants';
+import { FEATURE_TOGGLE_DEFAULTS, INITIAL_TOKENS, networkSettingsKey, PUSH_NOTIFICATION_FEATURE_TOGGLE, WALLET_SERVICE_FEATURE_TOGGLE, WALLET_SERVICE_REQUEST_TIMEOUT } from '../constants';
```

Comment:
```suggestion
import {
  FEATURE_TOGGLE_DEFAULTS,
  INITIAL_TOKENS,
  PUSH_NOTIFICATION_FEATURE_TOGGLE,
  WALLET_SERVICE_FEATURE_TOGGLE,
  WALLET_SERVICE_REQUEST_TIMEOUT,
  networkSettingsKey,
} from '../constants';
```

Reply:
I also suggested a change in the `networkSettingsKey` variable name in #333:

https://github.com/HathorNetwork/hathor-wallet-mobile/pull/333#discussion_r1355361239

Reply:
Done at 0fdb380582f73a4f8504f91c4c6282e21b7c1b42

Reply:
Name change at: c9c81e4559d8b63ede0b165a60e5af399809f613